shopify/resources/recurring_application_charge.py

```diff
@@ -9,6 +10,12 @@ def _get_first_by_status(resources, status):
 
 class RecurringApplicationCharge(ShopifyResource):
 
+    def usage_charges(self):
+        return shopify.UsageCharge.find(recurring_application_charge_id=self.id)
```

Comment:
__init__ is importing usage_charge as UsageCharge why "import shopify" and use the namespaced version?